pages/style-guide/html.md

```diff
@@ -4,74 +4,170 @@
 
```

Comment:
Add a Table of Contents, Basically add this to the top of your page:
`<script>{
	"title": "HTML Style Guide",
	"toc": true
}</script>`

You can see an example [here](https://raw.githubusercontent.com/jquery/contribute.jquery.org/master/pages/style-guide/js.md)